source/analysis/AM1DCalibrationAB.h

```diff
@@ -40,7 +40,7 @@ class AM1DCalibrationAB : public AMStandardAnalysisBlock
 	Q_OBJECT
 
 	Q_PROPERTY(QString dataName READ dataName WRITE setDataName)
-	Q_PROPERTY(QString normalizationName READ normalizationName WRITE setNormalizationName)
+	Q_PROPERTY(QString NormalizationName READ normalizationName WRITE setNormalizationName)
 	Q_PROPERTY(double energyCalibrationOffset READ energyCalibrationOffset WRITE setEnergyCalibrationOffset)
```

Comment:
Found a fixed one bug that only showed up when testing on an existing DB. changing the case of this name triggers a DB upgrade that fails because the name already exists (I wouldn't actually want a DB upgrade anyways - I can live with the capital), causing AM to exit with code 255. 

@dretrex, is that a bug in the DB upgrade method?  Should be be case smart?  Or maybe it should just be used by competent coders...